File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        list=[]
        e=0
        o=n
        for i in range(len(nums)):
            if i%2==0:
                list.append(nums[e])
                e+=1
            else:
                list.append(nums[o])
                o+=1
        return list

File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int arr[]=new int[2*n];
        int j=0, k=n;
        for(int i=0; i<arr.length; i++){
            if(i%2==0){
                arr[i]=nums[j++];
            }else{
                arr[i]=nums[k++];
            }
        }
        return arr;
    }
}